conanfile.py

```diff
@@ -19,21 +19,21 @@ class CubicInterpolationConan(ConanFile):
         "fPIC": [True, False],
     }
     default_options = {
-        "shared": False,
+        "shared": True,
         "fPIC": True
     }
 
     exports_sources = "*"
-    generators = "cmake"
+    generators = "cmake_find_package", "cmake_find_package_multi"
```

Comment:
Is there a reason why the `cmake_find_package_multi` generator is used although it's not used in the CMakeLists files?

Reply:
It is used. the `multi` generator creates the `Eigen3Config.cmake` and the one without multi produces the `FindBoost.cmake`, both are used and needed.

Reply:
Why not simply use the `FindEigen3.cmake` module generated by the `cmake_find_package` generator?

Reply:
Because `...Config.cmake` scripts are preferred to `Find....cmake` scripts.